### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#31] [Comments]
# index

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/index.html

**Comment by NicoleEditor** [comment]
I did 3 of the AWS Cloud Practitioner practice exams and they were very helpful. Thank you so much for making them available! <3

**Comment by sunmiller** [comment]
In the question,
Which tool can a non-AWS customer use to compare the cost of on-premises environment resources to AWS?

A. AWS Cost Explorer.
B. AWS Pricing Calculator.
C. AWS Budgets.
D. AWS TCO Calculator.

Aws pricing calculator is a tool that is part of AWS TCO Calculator but you have D as the correct answer.  Can you please explain.
